cfme/physical/overview.py

```diff
@@ -0,0 +1,32 @@
+from navmazing import NavigateToAttribute
+from widgetastic_manageiq import StatusBox
+from cfme.base.login import BaseLoggedInPage
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator
+from cfme.utils.appliance import Navigatable
+
+
+class PhysicalOverview(Navigatable):
```

Comment:
Required:
Need collection. we are not currently using this. 
Please refer this current design model for `Integration test`
https://cfme-tests.readthedocs.io/en/master/guides/collections.html

Reply:
Hi @digitronik, does it really need a collection as it's a view, not an entity?

Reply:
my bad

Reply:
As per offline conversation. Its simple overview page no need of `BaseEntity` like container `overview` page.

Reply:
This is an unnecessary object that the navigation destination has been registered against, and should not exist.

The navigation destination that brings the browser to the `PhysicalOverviewView` should be registered against the physical provider collection.  That's the best contextual collection for this page in the UI, as its an overview of all providers, and their collected stats.

I think there's no need for a new module here, and this could just be in `cfme/physical/provider/__init__.py - but if you'd like to separate this into its own module, I'm not opposed to that structure.  Just import `PhysicalProviderCollection` and use it for navigation class registration.